rust/src/lib.rs

```diff
@@ -120,3 +120,6 @@ pub mod lzma;
 pub mod util;
 pub mod ffi;
 pub mod feature;
+
+#[allow(unused_imports)]
+pub use suricata_lua_sys;
```

Comment:
This is how we keep Lua library into the resulting binary without actually using it.  Needs comment stating this.

Reply:
Not really. It would be hard around what includes to use, etc. The other option for vendoring is to pull the Lua source right in, in which case it would be a little easier. But my question, is to what value? Is the effort worth it?